Cargo.toml

```diff
@@ -19,9 +26,11 @@ categories = [
 ]
 
 [workspace.dependencies]
-hdp-core = { version = "0.2.5", path = "crates/core" }
+hdp-preprocessor = { version = "0.2.5", path = "crates/pre-processor" }
```

Comment:
why u specify versions here
if u increase version of workspace all packages will bump up automatically and this version spec will conflict no?

Reply:
makesense rm in here https://github.com/HerodotusDev/hdp/pull/108/commits/6c7a733baf5a4ceee39f54740cc508d6502df8f1